src/pages/iou/request/step/IOURequestStepScan/index.native.js

```diff
@@ -61,12 +62,24 @@ function IOURequestStepScan({
     const styles = useThemeStyles();
     const devices = useCameraDevices('wide-angle-camera');
     const device = devices.back;
+    const navigation = useNavigation();
 
     const camera = useRef(null);
     const [flash, setFlash] = useState(false);
     const [cameraPermissionStatus, setCameraPermissionStatus] = useState(undefined);
     const askedForPermission = useRef(false);
 
+    const [didCapturePhoto, setDidCapturePhoto] = useState(false);
```

Comment:
Without this part it will work for second/third tap, but if you keep tapping until the page transition then it will show the same error message as before.
As I explained in the issue, after the photo is taken navigation will kick off, using this code will block taking new photo while navigation is in progress.

Reply:
Reverted, tested with second/third tap and it works fine.

Could be a feature request: show animation once photo is taken, something like circular progress while the navigates to next step.